Sources/PodToBUILD/Lens.swift

```diff
@@ -108,7 +108,6 @@ public func .. <A, B, C>(lhs: Lens<A, B>, rhs: Lens<B, C>) -> Lens<A, C> {
  - returns: The composed lens.
  */
 infix operator >•>: MultiplicationPrecedence
-@available(*, deprecated)
 public func >•> <A, B, C>(lhs: Lens<A, B?>, rhs: Lens<B, C?>) -> Lens<A, C?> {
```

Comment:
I removed this deprecation since there doesn't seem to be any work related to dedicated to removing the warnings in quite some time and it makes the build output harder to grok

Reply:
We determined the operated made the code harder to grok, which is why it is deprecated.
Another possibility for devs that don’t like the warnings, is to turn them off locally. Well should get to removing this soon!

Reply:
I'll add it back and we can remove later